<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirCmdEditor</class>
 <widget class="QWidget" name="DirCmdEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>1170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>必填参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>目录或文件路径：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_arg_dirOrFilepath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_chooseDirpath">
            <property name="text">
             <string>选择目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_chooseFilepath">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>可选参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_resetOptions">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkbox_option_p">
          <property name="text">
           <string>(/p)对于输出结果列表，每次只显示一屏，按任意键显示下一屏。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_q">
          <property name="text">
           <string>(/q)显示所有者信息。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_w">
          <property name="text">
           <string>(/w)以宽格式显示列表，每行最多包含五个文件名或目录名称。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_d">
          <property name="text">
           <string>(/d)以与 /w 相同的格式显示列表，但文件按列排序。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_option_a">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>(/a)文件/文件夹属性筛选</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_d">
             <property name="text">
              <string>(:d)目录</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_h">
             <property name="text">
              <string>(:h)隐藏文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_s">
             <property name="text">
              <string>(:s)系统文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_l">
             <property name="text">
              <string>(:l)重解析节点（用于实现符号链接，卷的挂载点等）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_r">
             <property name="text">
              <string>(:r)只读文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_a">
             <property name="text">
              <string>(:a)准备好归档的文件（不常用）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_a_i">
             <property name="text">
              <string>(:i)禁止内容索引服务编制索引的文件（不常用）</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_option_o">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>(/o)指定输出结果排序方式</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_n">
             <property name="text">
              <string>(:n)按文件名字母排序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_e">
             <property name="text">
              <string>(:e)按扩展名字母排序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_g">
             <property name="text">
              <string>(:g)先显示文件夹</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_s">
             <property name="text">
              <string>(:s)按文件大小从小到大排序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_d">
             <property name="text">
              <string>(:d)按日期/时间先后排序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_o_reverse">
             <property name="text">
              <string>(-)逆序</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_option_t">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>(/t)指定显示或用来排序的时间字段类型</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="chkbox_option_t_c">
             <property name="text">
              <string>(:c)使用文件创建时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_t_a">
             <property name="text">
              <string>(:a)使用文件上次访问时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkbox_option_t_w">
             <property name="text">
              <string>(:w)上次文件写入时间</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_s">
          <property name="text">
           <string>(/s)列出指定目录和所有子目录中出现的指定文件名的每次匹配项。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_b">
          <property name="text">
           <string>(/b)显示目录和文件的裸列表，没有其他信息。/b 参数将覆盖 /w。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_l">
          <property name="text">
           <string>(/l)使用小写显示未排序的目录名称和文件名。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_n">
          <property name="text">
           <string>(/n)在屏幕最右侧显示带有文件名的长列表格式。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_x">
          <property name="text">
           <string>(/x)显示为 non-8dot3 文件名生成的短名称。显示与 /n 的显示相同，但短名称插入到长名称之前。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_c">
          <property name="text">
           <string>(/c)以文件大小显示千位分隔符。这是默认行为。使用 /-c 隐藏分隔符。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_4">
          <property name="text">
           <string>(/4)以四位数格式显示年份。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_r">
          <property name="text">
           <string>(/r)显示文件的备用数据流。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkbox_option_help">
          <property name="text">
           <string>(/?)显示帮助文档。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
